src/components/projects/projects-table/projects-table-filter/projects-table-filter.tsx

```diff
@@ -8,6 +8,7 @@ import { Button, Card, Dropdown, Form } from 'react-bootstrap';
 import styles from './projects-table-filter.module.css';
 import { User } from 'utils';
 import { fullNamePipe } from '../../../../shared/pipes';
+import { WbsElementStatus } from '../../../../utils/src';
```

Comment:
this should be added to the existing `utils` import, not a new import because using relative path imports for the `utils` package is not the proper way to access things inside the `utils` package.